dashboard/modules/node/node_head.py

```diff
@@ -319,6 +320,11 @@ def process_error(error_data):
             except Exception:
                 logger.exception("Error receiving error info from GCS.")
 
+    @routes.get("/api/v0/nodes")
+    async def get_nodes(self, req) -> aiohttp.web.Response:
+        nodes = await self._dashboard_head.gcs_state_aggregator.get_nodes()
```

Comment:
What's returned for node? Are we returning the node id as hex string?

Reply:
I believe it returns hex string. I will add tests